pkgs/build-support/php/builders/v2/build-composer-project.nix

```diff
@@ -11,12 +11,11 @@ let
     {
       php ? finalAttrs.php or toplevel.php,
       composer ? finalAttrs.php.packages.composer or toplevel.php.packages.composer,
-      composerLock ? finalAttrs.composerLock or null,
-      vendorHash ? finalAttrs.vendorHash or "",
-      composerNoDev ? finalAttrs.composerNoDev or true,
-      composerNoPlugins ? finalAttrs.composerNoPlugins or true,
-      composerNoScripts ? finalAttrs.composerNoScripts or true,
-      composerStrictValidation ? finalAttrs.composerStrictValidation or true,
+      composerLock ? null,
+      composerNoDev ? true,
+      composerNoPlugins ? true,
+      composerNoScripts ? true,
+      composerStrictValidation ? true,
```

Comment:
I'm not sure I understand the intention behind the diff here. Can you elaborate, please?